doc/release-notes/10909-datacite-oai-harvesting.md

```diff
@@ -0,0 +1,10 @@
+### OAI Harvesting from DataCite
+
+DataCite maintains an OAI server (https://oai.datacite.org/oai) that serves records for every DOI they have registered. There's been a lot of interest in the community in being able to harvest from them. This way, it will be possible to harvest metadata from institution X even if the institution X does not maintain an OAI server of their own, if they happen to register their DOIs with DataCite. One extra element of this harvesting model that makes it especially powerful and flexible is the DataCite's concept of a "dynamic OAI set": a harvester is not limited to harvesting the pre-defined set of ALL the records registered by the Institution X, but can instead harvest virtually any arbitrary subset thereof; any query that the DataCite search API understands can be used as an OAI set (!).
+
+A few technical issues had to be resolved in the process of adding this functionality so, as of this release it is being offered as somewhat experimental. Its beta version is nevertheless already in use at IQSS with seemingly satisfactory results.
```

Comment:
Aside re: our "experimental" label. There are groups who avoid these features assuming they might go away/who are surprised to hear that Harvard is using them and is unlikely to drop them completely. We might want to think about removing the 'experimental' language after one version, or changing to 'cutting edge' or something.

Reply:
Agree. Also, the label may be unnecessary in this case. Not that I'm 100% positive it's not going to fail for anyone; but I don't think it can cause any real damage either.

Reply:
I got rid of the "experimental" part and otherwise slightly refined the docs.